src/ExposureHistory/detail/ExposureActions.tsx

```diff
@@ -118,8 +124,8 @@ const RequestCallBackActions: FunctionComponent<RequestCallBackActionsProps> = (
       </GlobalText>
       <Button
         customButtonStyle={style.requestCallbackButton}
-        customButtonInnerStyle={style.requestCallbackButtonGradient}
-        onPress={navigateToCallbackForm}
+        customButtonInnerStyle={style.requestCallbackButtonInner}
+        onPress={handleOnPressRequestCallback}
         label={t("exposure_history.exposure_detail.speak_with_contact_tracer")}
```

Comment:
Small 🥔  here but is the renaming to be consistent with what the button text is or to the action? If it is should we rename the translation to be request callback or the action itself?